alpha/apps/kaltura/lib/batch2/kFlowManager.php

```diff
@@ -686,6 +686,7 @@ public function objectChanged(BaseObject $object, array $modifiedColumns)
 			&& $object->getStatus() == entryStatus::READY)
 		{
 			kFlowHelper::addDeliveryTagToEntry($object);
+			kFlowHelper::handleDelayedNotification($object);
```

Comment:
when entry becomes ready and there is a delayed job we release the job regardless of the notification template configuration 
1. what if we define different conditions later for notification templates - need to check if the notification template condition if it's indeed entry ready
2. what if we have multiple notification templates with delayed condition - need to distinguish between them